openff/toolkit/tests/test_molecule.py

```diff
@@ -3711,25 +3711,46 @@ def test_get_bond_between(self):
     @requires_rdkit
     def test_molecule_rings(self, smiles, n_rings):
         """Test the Molecule.rings property"""
-        assert (
-            n_rings == Molecule.from_smiles(smiles, allow_undefined_stereo=True).n_rings
+        mol = Molecule.from_smiles(smiles, allow_undefined_stereo=True)
+        assert n_rings == mol.get_n_rings()
+
+    def test_is_in_ring(self):
+        """
+        Test basic behavior of Atom.is_in_ring and Bond.is_in_ring API.
+        More extensive behavior testing is done in test_toolkits.py
+        """
+        molecule = Molecule.from_smiles(
+            "[H:9][c:3]1[c:2]([c:1]([c:6]([c:5]([c:4]1[H:10])[H:11])[H:12])[H:7])[H:8]"
         )
+        assert molecule.atoms[1].is_in_ring()
+        assert molecule.bonds[1].is_in_ring()
 
+    @requires_rdkit
+    @requires_openeye
+    @pytest.mark.parametrize(
+        "toolkit_wrapper", [RDKitToolkitWrapper, OpenEyeToolkitWrapper]
+    )
     @pytest.mark.parametrize(
-        ("smiles", "n_atom_rings", "n_bond_rings"),
+        ("smiles", "n_rings"),
         [
-            ("c1ccc2ccccc2c1", 10, 11),
-            ("c1ccc(cc1)c2ccccc2", 12, 12),
-            ("Cc1ccc(cc1Nc2nccc(n2)c3cccnc3)NC(=O)c4ccc(cc4)CN5CCN(CC5)C", 30, 30),
+            ("c1ccc2ccccc2c1", 2),
+            ("c1ccc(cc1)c2ccccc2", 2),
+            ("Cc1ccc(cc1Nc2nccc(n2)c3cccnc3)NC(=O)c4ccc(cc4)CN5CCN(CC5)C", 5),
         ],
     )
-    @requires_rdkit
-    def test_is_in_ring(self, smiles, n_atom_rings, n_bond_rings):
-        """Test Atom.is_in_ring and Bond.is_in_ring"""
+    def test_n_rings(self, smiles, n_rings, toolkit_wrapper):
```

Comment:
(blocking) This test should be removed because `n_rings` is being deleted from the API